src/code/code_8012EC80.c

```diff
@@ -547,9 +547,9 @@ void func_8012F1BC(s16 arg0);
 
 s16 func_8012F22C(s16 sceneIndex) {
```

Comment:
But that would require naming this system

Reply:
This function and file is already documented and addressed in https://github.com/zeldaret/mm/pull/534, so I'd rather it be left alone to prevent conflicts